org.eclipse.transformer/src/main/java/org/eclipse/transformer/action/impl/SignatureRuleImpl.java

```diff
@@ -41,10 +41,18 @@
 
 public class SignatureRuleImpl implements SignatureRule {
 
-	public SignatureRuleImpl(Logger logger,
+        public SignatureRuleImpl(Logger logger,
 
 		Map<String, String> renames, Map<String, String> versions, Map<String, BundleData> bundleUpdates,
 		Map<String, Map<String, String>> masterTextUpdates, Map<String, String> directStrings) {
+            this(logger, renames, versions, bundleUpdates, masterTextUpdates, directStrings, Collections.emptyMap());
+        }
```

Comment:
Do we really need to overload this constructor? We are not at release 1.0 yet, so just change the original constructor and all places which call it.

Reply:
Constructor removed.